### PR TITLE
feat: Add ubuntu summit 24 page

### PR DESCRIPTION
**Pull request by aaryanporwal**
## Done

- [x] Added website ubuntu summit 2024.

## QA

- Go to the below demo link and check if there are no regressions, check responsiveness on different devices (mobile, tablet, etc.)
- https://ubuntu-com-14316.demos.haus/summit

## Issue / Card

Fixes [WD-15079](https://warthogs.atlassian.net/browse/WD-15079)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-15079]: https://warthogs.atlassian.net/browse/WD-15079?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
Hi, as discussed, I will only comment on the most obvious things, the rest can wait until we redesign it next cycle.

- misaligned page body and navigation - compare with the ubuntu.com homepage to see how we usually align those.
image

- we do strips differently now - pllease have a look at the /core bubble on u.c for example: `.p-section>.row>hr+h2` instead of a p-strip
https://ubuntu.com/core/features

- example  of how we might do the speakers (imagine faces inside those grey boxes):
image

- no dark strips, or anything that changes the background edge to edge. Curiously, we often see people accelerate scrolling when coming across these, presumably because they remind them of ads:
image

- see how we do blog posts/ webinars etc - with an image and a multiple-of-25% layout, as explained [here ](https://www.figma.com/design/DZ7V6EFl18u72JLJrFDB2v/Design-system-guidelines-v.-0.1?node-id=897-9402&t=aEoKV1eN8wnCu5PR-1)
- https://ubuntu.com/desktop

**Comment by aaryanporwal** [comment]
> Hi, as discussed, I will only comment on the most obvious things, the rest can wait until we redesign it next cycle.
> 
> * misaligned page body and navigation - compare with the ubuntu.com homepage to see how we usually align those.
>   ...

Thank you Lyubo for the review and for sharing your insights. I'll address them as soon as possible. 

Also I learnt something really valuable here:
> presumably because they remind them of ads

That really explains why that section seemed a bit odd.

**Comment by aaryanporwal** [comment]
@lyubomir-popov Can you take another look. I've made the changes!

**Comment by lyubomir-popov** [comment]
- Hero: Can we please use h1, h2, h5for the headings:

image

- h1+h2 distance is not the same as in vanilla when I try in-browser. Can you pls update vanilla, and make sure you're not using the ubuntu.com local styles, as they currently override the distance between h1 andn h2 due to a bug

https://vanillaframework.io/docs/examples/base/headings/default?theme=light

video doesn't span the parent container (remove iframe 20px padding)

image

- pls use h5+p for speaker names:

image
-   logo width should be 104px not 200

image

**Comment by aaryanporwal** [comment]
@lyubomir-popov I've made the changes. Please take a look.

**Comment by anthonydillon** [comment]
@lyubomir-popov this ticket is blocked on your review and ultimately your design +1. The label should be switched to +1

**Review by lyubomir-popov — APPROVED** [review]

**Comment by anthonydillon** [comment]
@aaryanporwal why did you merge this? There is still code and QA reviews required.

**Comment by lyubomir-popov** [comment]
@anthonydillon I've approved it. There are things that I would do differently, if this project hadn't sidestepped the design team, as we've discussed previeously with the team, but for a page built without a design, I think it looks acceptable.

File: templates/summit/index.html [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+        </p>
+        <p>
+          <a href="https://events.canonical.com/event/51/registrations/"
+             class="p-button--positive">Register for free</a>
+        </p>
+      </div>
+      <div class="col-4 u-vertically-center">
+        <img src="https://assets.ubuntu.com/v1/9e91caf9-Canonical%20Ubuntu%20Summit%20Logo%202024%20with%20text.svg"
+             width="350"
+             alt="Ubuntu Summit 2024 Logo" />
```
Comment (anthonydillon) [inline templates/summit/index.html:63]: I would suggest using the image tag to optimise this image.

File: templates/summit/index.html [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+  </section>
+  <section class="p-section">
+    <div class="row">
+      <hr class="p-rule" />
+      <h2>Featured Speakers</h2>
+      <div class="col-2 p-card">
+        <div class="p-card__content">
+          <div class="p-card__image">
+            <img src="https://events.canonical.com/user/1666/picture-895558193"
+                 alt="Mark Jackels" />
```
Comment (anthonydillon) [inline templates/summit/index.html:97]: I would suggest using the image tag to optimise this image and lazy load them.

File: templates/summit/index.html [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+      <p>
+        We're introducing a new element to the Ubuntu Summit experience. This year, attendees can visit a number of community focused booths stationed throughout the main venue space.
+      </p>
+
+      <div class="p-logo-section has-misaligned-images">
+        <div class="p-logo-section__items">
+          <div class="p-logo-section__item">
+            <img src="https://assets.ubuntu.com/v1/d86e845a-Canonical_logo_2023.svg"
+                 width="104"
+                 alt="Canonical" />
```
Comment (anthonydillon) [inline templates/summit/index.html:262]: I would suggest using the image tag to optimise this image and lazy load them.

**Review by anthonydillon — COMMENTED** [review]
Some suggestions to use the image tag and there are no links to this page anywhere or in the sitemap therefore it can not be discovered by users or bots.

**Comment by anthonydillon** [comment]
The logo's look all over the place. Can these be updated to have the correct whitespace:
image

File: templates/summit/index.html (hunk 1 above)
Comment (aaryanporwal) [reply templates/summit/index.html:63]: Ah I see, you mean using jinja image tag?

`{{ image(url="https://assets.ubuntu.com/v1/d3039f38-MAAS-logo.svg",
                                alt="MAAS",
                                width="104",
                                height="42",
                                hi_def=True,
                                loading="auto") | safe}}`

**Comment by aaryanporwal** [comment]
@Sophie-32 
> * is there a full list of the community booths too?
No, these are all the community booths.

**Pull request merged**